### PR TITLE
change the prow configuration for `pingcap/tidb-engine-ext` repository.

### LINKED ISSUE #1066: change the prow configuration for `pingcap/tidb-engine-ext` repository.

### PR DESCRIPTION
**Pull request by wuhuizuo**
Fixes #1066

Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

### CODE REVIEW

File: prow/config/config.yaml [hunk 1]
```diff
@@ -562,6 +562,13 @@ branch-protection:
             release-8.1: *tidb-binlog-branch-pt
             # release-{{.ver}}: *tidb-binlog-branch-pt
 
+        tidb-engine-ext:
+          branches:
+            raftstore-proxy:
+              protect: true
+              required_status_checks:
+                contexts: [DCO, license/cla, tide]
```
Comment (wuhuizuo) [inline prow/config/config.yaml:570]:
```suggestion
                contexts: [tide]
```

**Review by purelind — APPROVED** [review]
/lgtm

**Pull request merged**